### ISSUE TITLE
v1.9.3 crashes with Webpack

### ISSUE THREAD

**Issue by codeheroics** [negomi/react-burger-menu#77]
v1.9.3 crashes at app load with Webpack.
I'm guessing this is due to the problems with Snap.svg which are mentioned in the code and refer to https://github.com/adobe-webplatform/Snap.svg/issues/341 , as https://github.com/negomi/react-burger-menu/commit/66388f57fd331df7e4fd62af94c89c854dcf0872 added `import Snap from 'snapsvg';` to two files of react-burger-menu

**Comment by snakeyak** [comment]
Same problem here. Here's the error I'm getting:
```
/Users/xxx/node_modules/snapsvg/dist/snap.svg.js:439
}(window || this, function (window, eve) {
  ^
ReferenceError: window is not defined
    at Object.<anonymous> (/Users/xxx/node_modules/snapsvg/dist/snap.svg.js:439:3)
```
1.9.2. works fine

**Comment by iNikNik** [comment]
@snakeyak same

**Comment by wongteam** [comment]
@snakeyak same

**Comment by negomi** [comment]
Ah, just from the import? :/

Stick with 1.9.2 for now and I'll look into it as soon as I can. Thanks for flagging the issue.

**Comment by negomi** [comment]
Fixed in v1.9.5
